Sources/SwiftIdentifier.swift

```diff
@@ -78,5 +74,19 @@ func swiftIdentifier(from string: String,
       return ""
     }
   })
-  return prefix + mappedParts.joined(separator: replacement)
+
+  let result = mappedParts.joined(separator: replacement)
+  return prefixWithUnderscoreIfNeeded(string: result, forbiddenChars: exceptions)
+}
+
+func prefixWithUnderscoreIfNeeded(string: String,
```

Comment:
I'm not a fan of that function name, because it doesn't tell when it's needed — it doesn't tell that it makes sense only in the context of a Swift identifier.
But I have to admit that I'm not inspired either for a better name, so… ¯\_(ツ)_/¯